test/utility/fragments.ts

```diff
@@ -360,17 +385,50 @@ export const languageEngagement = gql`
         beginFiscalYear {
           value
         }
-        ethnologueName {
+        isDialect {
```

Comment:
This language object could reuse the language fragment.
```gql
language {
  ...language
}
```
Then add the fragment to the end of the string
```ts
export const languageEngagement = gql`
  fragment languageEngagement on LanguageEngagement {
    ...
  }
  ${language}
`;
```